resolversrv/views.py

```diff
@@ -0,0 +1,470 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import traceback
+import json
+
+from flask import current_app, request, Blueprint
+from flask_discoverer import advertise
+from flask import Response
+from requests.utils import quote
+
+from resolversrv.utils import get_records
+
+
+bp = Blueprint('resolver_service', __name__)
+
+class LinkRequest():
+
+    baseurl = 'https://ui.adsabs.harvard.edu/#abs'
```

Comment:
i don't see it being overriden elsewhere; user's may need to specify different baseurl values (in local_config.py)

Reply:
ok so I understand that for example we have solr address and it could get overwritten while in production. but how do you tell the deployer this is one of these variables that can be overwritten.

Reply:
moved to config, added a comment for the deployer that depending on where it is deployed this needs to be updated.